src/main/java/Duke.java

```diff
@@ -1,10 +1,209 @@
+                    taskings.add(new Deadline(line, deadlineDate, isDone));
+                    System.out.println("Got it. I've added this task:\n  " + taskings.get(itemNumber).toString() + "\nNow you have " + (itemNumber + 1) + " tasks in the list.");
+                    itemNumber++;
+
+                } else if (startsWithTodo) { //todo command given
+//                    list[itemNumber] = new Todo(line.substring(5), isDone);
+                    taskings.add(new Todo(line.substring(5), isDone));
+                    System.out.println("Got it. I've added this task:\n  " + taskings.get(itemNumber).toString() + "\nNow you have " + (itemNumber + 1 )+ " tasks in the list.");
+                    itemNumber++;
+
+                } else if (startsWithEvent) { //event command given
+                    indexOfBackslash = line.indexOf("/");
+                    deadlineDate = line.substring(indexOfBackslash + 4);
+                    line = line.substring(6, indexOfBackslash - 1);
+                    taskings.add(new Event(line, deadlineDate, isDone));
+//                    list[itemNumber] = new Event(line, deadlineDate, isDone);
+//                    System.out.println("Got it. I've added this task:\n  " + list[itemNumber].toString() + "\nNow you have " + itemNumber + " tasks in the list.");
+                    System.out.println("Got it. I've added this task:\n  " + taskings.get(itemNumber).toString() + "\nNow you have " + (itemNumber + 1) + " tasks in the list.");
+                    itemNumber++;
+
+                } else if (startsWithDelete) {
+
+                    taskNumber = Integer.parseInt(line.substring(7));
+                    System.out.println("Noted. I've removed this task: \n  " + taskings.get(taskNumber - 1).toString() + "\nNow you have " + (itemNumber - 1) + " tasks in the list.");
+                    taskings.remove(taskNumber - 1);
+                    itemNumber--;
+
+                } else if (!startsWithBye) {
+                    throw new DukeExceptions();
+                }
+
+
+            } catch (IndexOutOfBoundsException e) {
+                if (startsWithTodo) {
+                    System.out.println(TODO_ERROR);
+                } else if (startsWithEvent) {
+                    System.out.println(EVENT_DESCRIPTION_ERROR);
+                } else if (startsWithDeadline) {
+                    System.out.println(DEADLINE_DESCRIPTION_ERROR);
+                }
+            } catch (NullPointerException e) {
+                System.out.println(DONE_WRONG_NUMBER_ERROR);
+            } catch (NumberFormatException e) {
+                System.out.println(DONE_ERROR);
+            } catch (DukeExceptions e) {
+                System.out.println(GENERAL_ERROR);
+            }
+        }
+
+        System.out.println("Bye. Hope to see you again soon!\n");
```

Comment:
Since the message is static, perhaps the sentence could be stored as a constant variable instead